tests/getMediaUrl.test.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+import { ServerlessCallback } from '@twilio-labs/serverless-runtime-types/types';
+import axios from 'axios';
+import { handler as getMediaUrl, Event } from '../functions/getMediaUrl';
+
+import helpers, { MockedResponse } from './helpers';
+
+jest.mock('@tech-matters/serverless-helpers', () => ({
+  ...jest.requireActual('@tech-matters/serverless-helpers'),
+  functionValidator: (handlerFn: any) => handlerFn,
+}));
+
+jest.mock('axios', () => ({
+  request: jest.fn(),
+}));
```

Comment:
Stephen, I missed the original Serverless code, but I think we should not be using `axios` anymore. We're moving away from `axios` to use node's `fetch`.

This test is very good that it made me realize that we should change the Serverless code to use `fetch`. Can you do that as part of this PR?

Reply:
@murilovmachado - I have updated the code and test as requested.